packages/lib/person/service.ts

```diff
@@ -264,50 +277,36 @@ export const getOrCreatePersonByUserId = async (userId: string, environmentId: s
         select: selectPerson,
       });
 
-      if (personPrisma) {
-        return personPrisma;
-      } else {
-        // Create a new person with the userId attribute
-        const userIdAttributeClass = await getAttributeClassByName(environmentId, "userId");
-
-        if (!userIdAttributeClass) {
-          throw new ResourceNotFoundError(
-            "Attribute class not found for the given environment",
-            environmentId
-          );
-        }
+      if (personWithUserIdAttribute) {
```

Comment:
I would like to update the person directly here if he still has a userId in the attributes. So I would like to check if userId is set in attributes and if so, set the userId in person.userId and remove the attribute